### ISSUE TITLE
Confusion about low frequency loss

### ISSUE THREAD

**Issue by minzhang-1** [TomTomTommi/HiNet#8]
Hi, the low frequency loss in your code seems to be different from what you describe in the paper. You only select the first image in the batch, each with 12 channels to calculate loss. (If so, it will has no difference with the guide loss.) And the lambda is also not 10. Could you please explain a little bit?

**Comment by TomTomTommi** [comment]
Thanks for your careful observation, maybe it happened during polishing. I have fixed this problem with a "narrow" operation. But I haven't re-run the code, so if there exists any more bugs, please let me know. 
As for the hyper-parameter lambda, the setting depends on the situation. You could set as 1 in the pre-stage and just fine-tune this hyper-parameter to make the results of cover and secret images balance.
